### ISSUE TITLE
body encoding

### ISSUE THREAD

**Issue by manelclos** [Rhumbix/django-request-logging#28]
I hit a problem using Python 3 and sending an image file to the server:

```
  File "/data/project/venv/lib/python3.5/site-packages/request_logging/middleware.py", line 132, in _log_multipart
    body_str = body if isinstance(body, str) else body.decode()
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xff in position 573: invalid start byte
```

There should be a global try-except to avoid any APP crash at all due to request-logging.

**Comment by tclancy** [comment]
Which version are you using? That looks like it was fixed in 0.5.3.

**Comment by manelclos** [comment]
Local devel environment using django-request-logging==0.5.3, fails.

On the production server all is ok with django-request-logging==0.5.2.

Downgraded local environment to 0.5.2 and it is not failing now, just the post info:
POST /app/api/v1/photos/

**Comment by tclancy** [comment]
Could you submit a test case? I'm guessing the `if isinstance(body, str)` test doesn't work well between Python 2 and 3 since `str` is the only string type in 3 but in 2 we should have `basestring` there (I think). Or maybe the test doesn't make sense in Python 3.

**Comment by tclancy** [comment]
Do you mind grabbing the newest copy from here to see if #23 fixed this issue?

**Comment by tclancy** [comment]
And to see if #30 fixed the issue?

**Comment by Sundava** [comment]
It seems like the issue is not fixed for me, using version 0.6.5 on python 2.7.12

" UnicodeDecodeError at /api/cats/314/
'ascii' codec can't decode byte 0xff in position 192: ordinal not in range(128)"

I added a preview from Sentry to show the breadcrumbs :

![problems](https://user-images.githubusercontent.com/7022870/46793108-36c33380-cd45-11e8-85f1-1a9b4c812081.png)

It seems like 'body' is of type str when the function is called, which means body.decode() is never called at the line 
`            body_str = body if isinstance(body, str) else body.decode()
`

Manually editing the code to force the `body.decode()` call seems to fix the problem

Does this project still support Python 2.7 ?

**Comment by tclancy** [comment]
We do still support 2.7. Can you provide a PR for this and then I will release a new version? There are a few other improvements that have been merged in since the last release.
